src/ponyc/main.c

```diff
@@ -118,6 +120,9 @@ static void usage()
     "  --stats         Print some compiler stats.\n"
     "\n"
     "Debugging options:\n"
+    "  --verbose       Verbosity level: 0: quiet, 1: default, \n"
```

Comment:
I see that `-v` is already taken for `--version`, but how about allowing `-V` as short for `--verbose`?